waku/factory/node_factory.nim

```diff
@@ -144,9 +144,9 @@ proc setupProtocols(
 
     peerExchangeHandler = some(handlePeerExchange)
 
+  let shards =
+    conf.contentTopics.mapIt(node.wakuSharding.getShard(it).expect("Valid Shard"))
   if conf.relay:
-    let shards =
-      conf.contentTopics.mapIt(node.wakuSharding.getShard(it).expect("Valid Shard"))
     let pubsubTopics = conf.pubsubTopics & shards
 
     debug "Shards created from content topics",
```

Comment:
I think we should move this log to be right below

https://github.com/waku-org/nwaku/blob/11b575d3f94de25b38e7a82f5809b7efdeac7460/waku/factory/node_factory.nim#L147-L148